core/model/modx/modtemplatevar.class.php

```diff
@@ -443,6 +442,7 @@ public function getRender($params,$value,array $paths,$method,$resourceId = 0,$t
                 /* 2.1< backwards compat */
                 $renderFile = $path.$type.'.php';
                 if (file_exists($renderFile)) {
+                    $this->xpdo->deprecated('2.2.0', '', '<2.2 style template variable flat render file ' . $renderFile . ', for TV ' . $this->get('name'));
```

Comment:
Don't include the deprecated since version twice and add a message, who has to fix that issue:
```
$this->xpdo->deprecated('2.2.0', '', 'Using old style template variable flat render file ' . $renderFile . ', for TV ' . $this->get('name') . '. Please contact the author of the template variable type to change this code part. );
```

Reply:
Are you happy with the updated message @Jako?